addons/sourcemod/scripting/gokz-hud/tp_menu.sp

```diff
@@ -197,14 +197,23 @@ static void TPMenuAddItemCheckpoint(KZPlayer player, Menu menu)
 	{
 		Format(display, sizeof(display), "%s #%d", display, player.CheckpointCount);
 	}
-	
-	if (player.CanMakeCheckpoint || !gB_DynamicMenu[player.ID])
-	{
-		menu.AddItem(ITEM_INFO_CHECKPOINT, display, ITEMDRAW_DEFAULT);
-	}
-	else
+	switch (gI_DynamicMenu[player.ID])
```

Comment:
Why are all of those checks `switch`es instead of `if`s if they only have 2 cases each?

Reply:
Whoops. Previously that was going to switch between three possible values of the dynamic menu but that was unnecessary. I will change that.